src/tools/linter-rules.md

```diff
@@ -30,13 +30,15 @@ To avoid the need to individually select compatible linter rules,
 consider starting with a linter rule set,
 which the following packages provide:
 
-[lints]({{site.pub-pkg}}/lints)
-: The rule sets that the Dart team encourages using. 
+[lints][]
+: The rule sets that the Dart team encourages using
+  for command line and server applications. 
+  For Flutter programs, see [flutter_lints][].
```

Comment:
This doesn't seem quite right, since we encourage these lints for _all_ Dart code, and command line and server aren't all the non-Flutter use cases for Dart. Also not sure we need a link to `flutter_lints`, since that's the next item.

Maybe:
```suggestion
: The rule sets that the Dart team encourages using.
  If you're writing Flutter code,
  use the `flutter_lints` rule set,
  which builds on `lints`.
```

Reply:
That makes a lot of sense with `flutter_lints` building on them. This seems nicer and less cluttered, thanks!